content/en/docs/reference/command-line-tools-reference/kube-scheduler-config.v1beta1.md

```diff
@@ -0,0 +1,1068 @@
+
+
+
+
+
+---
+title: kubescheduler.config.k8s.io/v1beta1
+content_type: tool-reference
```

Comment:
Aside (shouldn't block merge): we could set `description` or `overview` or both, here.

Reply:
Filed an issue (https://github.com/kubernetes-sigs/reference-docs/issues/211) in case this ball is dropped.